<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestIulianaClass</class>
 <widget class="QMainWindow" name="TestIulianaClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>836</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TestIuliana</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>801</width>
      <height>491</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="RightLayout">
       <item>
        <widget class="QComboBox" name="comboBox">
         <item>
          <property name="text">
           <string>Detailed</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Simplified</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="adminButtons">
         <item>
          <widget class="QPushButton" name="addButton">
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="removeButton">
           <property name="text">
            <string>Remove</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="updateButton">
           <property name="text">
            <string>Update</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="undoButton">
         <property name="text">
          <string>Undo</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="trenchCoatList"/>
       </item>
       <item>
        <widget class="QLabel" name="sizeLabel">
         <property name="text">
          <string>Size:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="sizeEdit"/>
       </item>
       <item>
        <widget class="QLabel" name="colourLabel">
         <property name="text">
          <string>Colour:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="colourEdit"/>
       </item>
       <item>
        <widget class="QLabel" name="priceLabel">
         <property name="text">
          <string>Price:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="priceEdit"/>
       </item>
       <item>
        <widget class="QLabel" name="quantityLabel">
         <property name="text">
          <string>Quantity:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="quantityEdit"/>
       </item>
       <item>
        <widget class="QLabel" name="photoLabel">
         <property name="text">
          <string>Photo:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="photoEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="goToAdmin">
         <property name="text">
          <string>&lt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="goToUser">
         <property name="text">
          <string>&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="LeftLayout">
       <item>
        <widget class="QListWidget" name="seeCurrent"/>
       </item>
       <item>
        <widget class="QComboBox" name="filterBySize">
         <item>
          <property name="text">
           <string>Size S</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Size M</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Size L</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Size XL</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Size XXL</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="nextTrenchCoat">
         <property name="text">
          <string>Next trench coat</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="buyCart">
           <property name="text">
            <string>Buy shopping bag</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="addToCart">
           <property name="text">
            <string>Add item to shopping bag</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListWidget" name="seeShoppingBag">
         <property name="enabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="putFilename">
         <item>
          <widget class="QLabel" name="filenameLabel">
           <property name="text">
            <string>Filename path:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="filename"/>
         </item>
         <item>
          <widget class="QPushButton" name="saveOrder">
           <property name="text">
            <string>Save shopping bag  to file</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="openShopping">
           <property name="text">
            <string>Open File</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>836</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="testiuliana.qrc"/>
 </resources>
 <connections/>
</ui>
